src/test/java/net/snowflake/client/jdbc/MockConnectionTest.java

```diff
@@ -803,5 +807,10 @@ public SFBaseFileTransferAgent getFileTransferAgent(String command, SFBaseStatem
               : SFBaseFileTransferAgent.CommandType.DOWNLOAD;
       return new MockSFFileTransferAgent(fileMap, "fileName", commandType);
     }
+
+    @Override
+    public String getBindStageName() {
+      return null;
```

Comment:
Maybe give it a dummy name `foo`? Having `null` might cause unexpected NPE during testing.

Reply:
Good call. But after I make `getBindStageName()` have a `default` implementation, it will fix this issue (and I can remove the override).